### ISSUE TITLE
HTML report doesn't escape closing `</script>` tags

### ISSUE THREAD

**Issue by llimllib** [GoogleChrome/lighthouse#996]
Steps to reproduce:

1. Visit healthcare.gov and go through healthcare application until you reach `/marketplace/auth/enroll/consumers/?a=[application_id]&cache=true&t=[state]#/`
2. Run lighthouse
3. See this result:

![2016-11-21 12-59-15](https://cloud.githubusercontent.com/assets/7150/20494350/7e60dc1c-afea-11e6-88e5-69eac73cb6b1.png)

I've attached the entirety of the report html at: https://gist.github.com/llimllib/931b448c9ec5cf7477959d9022210b18 . As you can see, line 1699 appears to be perfectly syntactically valid. Could this instead be a Chrome bug?

Google Chrome Version 54.0.2840.98 (64-bit)
MacOS Sierra
Lighthouse 1.2.0

It's interesting to note that the developer tools stops syntax highlighting properly at line 1699, which really makes me think this might be a parser error (!):

![2016-11-21 13-05-54](https://cloud.githubusercontent.com/assets/7150/20494536/640db97e-afeb-11e6-87d7-1294bd115ae4.png)

**Comment by wardpeet** [comment]
thanks for reporting it's a lighthouse bug

**Comment by ebidel** [comment]
We need to escape the `</script>`. I can take this.

**Comment by jarzuaga** [comment]
I get something similar running lighthouse against http://www.univision.com

**Comment by ebidel** [comment]
Looks like this is an interesting case with using `document.write` with a script tag:

The arg to the call site usage is the script tag string:

`document.wrote("<script src='//assets.healthcare.gov/resources/libs/optimizely/optimizely.js'></script>")`

We embed the original results object in the HTML report like so:

```
<script>
  // expose the original results
  self.lhresults = {{{ lhresults }}};
</script>
```

Unfortunately, when you do this, the `</script>` closes the `<script>` and breaks the page! This is only a problem for the HTML report and embedding JSON with a script.

We can fix this two ways:  
- remove the `lhresults` in the page. Although I think having the original object in the page is helpful for debugging and knowing the full results. This bug is case in point :)
- escape `</script>` in the HTML report. 

Going to go the latter.

**Comment by llimllib** [comment]
Surely the developer tools highlighting is broken there too - that's a perfectly valid javascript object? Should I report that as a bug on Chrome?

**Comment by ebidel** [comment]
Yea. Might want to look for an open bug first :)

**Comment by llimllib** [comment]
And thanks for the quick response and fix! 💯

**Comment by brendankenny** [comment]
fixed n #1003

FWIW, DevTools has always had some issues with syntax highlighting very long text input. I would be very surprised if there wasn't an existing bug and/or it was working as intended for performance reasons.
